db/seeds/utils.js

```diff
@@ -20,3 +23,15 @@ exports.formatComments = (comments, idLookup) => {
 		};
 	});
 };
+
+// exports.idNotExist = (table, column, value) => {
```

Comment:
And remove this please! You don't have to delete stuff like this, but just make sure it's not included in your commits.